webcomponents/tool/src/main/frontend/packages/sakai-grader/src/sakai-grader-rendering-mixin.js

```diff
@@ -589,6 +589,12 @@ export const graderRenderingMixin = Base => class extends Base {
                       <div class="grader-history-block">${unsafeHTML(this._submission.history.grades)}</div>
                     </div>
                   ` : nothing }
+                  ${this._submission.history.feedback_comment ? html`
```

Comment:
Should be this._submission.history.feedbackComment?

Reply:
Aghh, done...